cmd/root.go

```diff
@@ -42,6 +43,7 @@ func Execute(version FullVersion) error {
 		DropTagNamePrefix: *dropPrefixFlag,
 		PrereleaseSuffix:  *prereleaseSuffixFlag,
 		Format:            *formatFlag,
+		PrereleasePrefix:  *prereleasePrefixFlag,
```

Comment:
For sake of consistent ordering maybe move it up one line (suffix, prefix, format) to match the ordering of the flags above

Reply:
I changed order both in caller (Execute method,) and in the type definition.